desktop/src/app/components/resources/grid-list/grid-item.scss

```diff
@@ -39,6 +39,16 @@ grid-item {
     }
   }
 
+  .grid-resource-shortdescription {
```

Comment:
This should be written as ".grid-resource-short-description" (to reflect the upper case letter in "shortDescription").